data/maps/SlateportCity_PokemonCenter_1F/map.json

```diff
@@ -12,7 +12,7 @@
   "allow_running": false,
   "show_map_name": false,
   "battle_scene": "MAP_BATTLE_SCENE_NORMAL",
-  "connections": null,
```

Comment:
These map file changes seem unrelated to the PR?

Reply:
Yep, sorry. These were caused by opening porymap and were removed in a later commit.